dockers/docker-fpm-frr/frr/zebra/zebra.conf.j2

```diff
@@ -7,8 +7,13 @@
 {% endblock banner %}
 !
 {% block fpm %}
+{% if ( ('nexthop_group' in FEATURE) and ('state' in  FEATURE['nexthop_group']) and
+        (FEATURE['nexthop_group']['state'] == 'enabled') ) %}
+fpm use-next-hop-groups
```

Comment:
can you follow example here?

https://github.com/sonic-net/SONiC/blob/master/doc/BGP/BGP-supress-fib-pending.md#632--device_metadata

Reply:
@lguohan Moved under device metadata following the example.